xcodeproj/internal/xcode_schemes.bzl

```diff
@@ -6,6 +6,17 @@ load(":xcode_schemes_internal.bzl", "xcode_schemes_internal")
 
 _DEFAULT_BUILD_CONFIGURATION_NAME = "Debug"
 
+def _focus_actions(build_targets, actions):
+    # TODO:
```

Comment:
FYI. It looks like you have an errant TODO comment.

Reply:
Yes, @brentleyjones mentioned that he will be helping with this. (https://github.com/buildbuddy-io/rules_xcodeproj/pull/1047/files#r964738570)